SonarOnPull/Program.cs

```diff
@@ -10,6 +10,33 @@ class Program
     {
         static void Main(string[] args)
         {
+            Console.WriteLine("Result:" + AddNumbers(3,4,52,3));
+            Console.WriteLine("Result:" + AddAndSubstractNumbers(3, 10, 52,55));
+        }
+
+        static int AddNumbers(params int[] n)
+        {
+            int result = 0;
+            foreach(int num in n)
+            {
+                result += num;
+            }
+            while(true) { }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://138.68.127.131:9000/coding_rules#rule_key=csharpsquid%3AS108)